controllers/articles.controller.js

```diff
@@ -0,0 +1,16 @@
+const { selectArticleById } = require('../models/articles.model');
+
+
+exports.getArticleById = (req, res, next) => {
+  const article_id = Number(req.params.article_id)
```

Comment:
You also don't need to convert the article_id to a number. As before, PSQL will throw an error if it's not a number.